sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateAsyncClientTest.java

```diff
@@ -500,20 +501,20 @@ public void cancelCertificateOperation(HttpClient httpClient, CertificateService
         createCertificateAsyncClient(httpClient, serviceVersion);
 
         cancelCertificateOperationRunner((certName) -> {
-            PollerFlux<CertificateOperation, KeyVaultCertificateWithPolicy> certPoller =
-                certificateAsyncClient.beginCreateCertificate(certName, CertificatePolicy.getDefault());
+            // Had to use a sync poller as the async version of this test was very flaky.
+            // TODO (vcolin7): Figure out the best way to cancel an operation consistently.
+            SyncPoller<CertificateOperation, KeyVaultCertificateWithPolicy> certPoller =
```

Comment:
We should not use sync poller in async client test. Let's investigate why async poller is flaky.